app/helpers/grant-details.js

```diff
@@ -1,5 +1,5 @@
 const MIN_GRANT = 35000
-const MAX_GRANT = 500000
```

Comment:
@salfedev this grant is capped at 50000 so this is correct

Reply:
OK, I created a separate const for project cost validation @diya2214